### DISCUSSION TITLE
Support for RWX

### DISCUSSION THREAD

**Discussion by anthonymac1** [topolvm/topolvm#925] [Q&A] [answered]
Hi,

Is there any plan to support RWX in TopoLVM.

**Comment by llamerada-jp** [comment, marked as answer]
Hi,
TopoLVM just provides local volume for PV, so it isn't network storage. The volumes provided by TopoLVM cannot be mounted on different nodes. TopoLVM cannot support RWX access mode by this mechanism.

**Reply by anthonymac1** [reply]
Hi, 

I didn't intend to useTopoLVM across multiple nodes.  I wanted to keep my PVC templates the same where the PVC is created in one deployment on a single node, and for the other deployment it can be across multiple nodes.   I didn't want to have RWO PVC on the single node deployment and RWX PVC on multiple node deployment.

**Reply by llamerada-jp** [reply]
We do not make such changes because returning a capability that is not correct will result in unexpected behavior.
